docs/model_zoo.md

```diff
@@ -18,7 +18,8 @@ To propose a model for inclusion, please submit a [pull request](https://github.
 | Faster RCNN | Object Detection | COCO | 148 MB | [.mar](https://torchserve.pytorch.org/mar_files/fastrcnn.mar) | [persons.jpg](../examples/object_detector/persons.jpg) |Eager|
 | MASK RCNN | Object Detection | COCO | 158 MB | [.mar](https://torchserve.pytorch.org/mar_files/maskrcnn.mar) | [persons.jpg](../examples/object_detector/persons.jpg) |Eager|
 | Text classifier | Text Classification | AG_NEWS | 169 MB | [.mar](https://torchserve.pytorch.org/mar_files/my_text_classifier_v2.mar) | [sample_text.txt](../examples/text_classification/sample_text.txt) |Eager|
-| FCN Resenet 101 | Image Segmentation | COCO | 193 MB | [.mar](https://torchserve.pytorch.org/mar_files/fcn_resnet_101.mar) | [persons.jpg](../examples/image_segmenter/fcn/persons.jpg) |Eager|
+| FCN ResNet 101 | Image Segmentation | COCO | 193 MB | [.mar](https://torchserve.pytorch.org/mar_files/fcn_resnet_101.mar) | [persons.jpg](../examples/image_segmenter/fcn/persons.jpg) |Eager|
+| DeepLabV3 ResNet 101 | Image Segmentation | COCO | 217 MB | [.mar](https://torchserve.s3.amazonaws.com/mar_files/deeplabv3_resnet_101.mar) | [persons.jpg](https://github.com/pytorch/serve/blob/master/examples/image_segmenter/persons.jpg) |Eager|
```

Comment:
mar file link should be: https://torchserve.pytorch.org/mar_files/deeplabv3_resnet_101.mar

Reply:
Done! 👍🏻